src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -105,15 +114,16 @@ public function load(array $configs, ContainerBuilder $container)
         // default in the Form and Validator component). If disabled, an identity
         // translator will be used and everything will still work as expected.
         if ($this->isConfigEnabled($container, $config['translator']) || $this->isConfigEnabled($container, $config['form']) || $this->isConfigEnabled($container, $config['validation'])) {
-            if (!class_exists('Symfony\Component\Translation\Translator') && $this->isConfigEnabled($container, $config['translator'])) {
+            $container->addResource(new ClassExistenceResource(Translator::class));
```

Comment:
Why should we track classes for throwing cases? Isn't the container *not* dumped when an exception is thrown? Thus there is nothing to track? (same below)